config/initializers/activity_logs_subscription.rb

```diff
@@ -0,0 +1,6 @@
+ActiveSupport::Notifications.subscribe "process_action.action_controller" do |name, started, finished, unique_id, data|
+  args = data.slice(:controller, :action, :format, :method, :path, :status, :current_user_id)
+  args[:payload] = data[:params].except(:action, :controller)
+
+  ActivityLogsEventHandler.new(args).process
```

Comment:
@radinreth please consider making ActivityLogEventHandler as a singleton class where it needs to instantiate only one time then:

log = LogParser.parse(args)
ActivityLogger.log(log)

use term Logger is better than Handler as normally handler have to do a kind of subscribe event and then handle/dispatch event but not process